Apps/Database/src/Migrations/20200817213421_AddRatingsTable.cs

```diff
@@ -0,0 +1,54 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+// <auto-generated />
+#pragma warning disable CS1591
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HealthGateway.Database.Migrations
+{
+    public partial class AddRatingsTable : Migration
```

Comment:
Ratings --> Rating for here and the name of the migration scripts (20200817213421_AddRatingsTable.cs  and .designer..) as well if you don't mind to change :-)

Reply:
I don't think is necessary, we have other names like that for the migrations

Reply:
no need to change per our rocket chat :)